### PR TITLE
Example code in ui

### PR DESCRIPTION
**Pull request by coffee-cup**
- Use babel generator to get source for ast node
- Use prettier to format the source
- Show source for examples in UI

image

image

image

### CODE REVIEW

File: packages/server/src/generate.ts [hunk 1]
```diff
@@ -175,8 +163,25 @@ const generateStylesArray = (files: File[], searchDir: string): string =>
...
+      };
+
+      return `{forComponent: ${
+        defaultExport.id
+      }, examples: [\n    ${namedExports
+        .map(generateSingleExample)
+        .join(",\n    ")}\n  ]}`;
+    })
+    .join(",\n  ");
+// files.map(({id}) => id).join(",\n  ");
```
Comment (andrejak) [inline packages/server/src/generate.ts]: Unnecessary comment at the end?

File: packages/ui/src/App/context.tsx [hunk 2]
```diff
@@ -15,26 +23,21 @@ const ThemeProvider =
...
   ReactRouterDOM != null ? ReactRouterDOM.MemoryRouter : React.Fragment;
 
-const findExamplesForComponent = (c: Component) => {
-  const exampleFileExport = userImport.examples.filter(
-    (ex: any) => ex.default !== null && ex.default === c.component,
+const exampleImports: ExampleImport[] = userImport.examples;
+
+const findExamplesForComponent = (c: Component): Example[] => {
+  const exampleFileExport = exampleImports.filter(
+    ex => ex.forComponent !== null && ex.forComponent === c.component,
```
Comment (andrejak) [inline packages/ui/src/App/context.tsx]: Should maybe examples with null properties be filtered out earlier on? would an example with null forComponent ever be shown?

**Comment by andrejak** [comment]
I'm not sure about the background colour (and the purplish undertone?) of the source code - maybe we should distinguish it more from the bg color, or perhaps not at all. It's not very important though, and if I'm the only one feel free to ignore it.

File: packages/ui/src/App/context.tsx (hunk 2 above)
Comment (coffee-cup) [reply packages/ui/src/App/context.tsx]: Oops, this should be `!= null` instead. The `forComponent` can be undefined if the user does not have a default export from the examples file.

**Comment by coffee-cup** [comment]
@andrejak I don't really like the purple highlighting either. I was just going with the default theme. Probably a good idea to change it though. Do you think it should be a dark or light theme?

**Comment by coffee-cup** [comment]
What do you think of 

image

**Comment by andrejak** [comment]
I like the theme, but do you think the border is necessary? I think it could look nicer without, and it's obvious what the code is refering too. (Atm I feel like there's too many borders, and the radius is a bit different too)

**Comment by coffee-cup** [comment]
I like it without the border

image

**Pull request merged**